cfme/tests/automate/test_buttons.py

```diff
@@ -35,36 +33,101 @@ def dialog():
     yield service_dialog
 
 
+# IMPORTANT: This is a canonical test. It shows how a proper test should look like under new order.
 @pytest.mark.sauce
 @pytest.mark.tier(2)
-def test_button_group_crud(request):
+def test_button_group_crud(request, soft_assert):
```

Comment:
You add soft_assert here, but then make standard asserts, are these caught by soft_assert? I though you had to run them with the soft_assert?